azure.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+###
+# Configuration
+###
+
+# The path of the Azure CLI executable
+AZURE_CMD="/home/ryan/.bin/az"
```

Comment:
Would it make more sense for values like these to be passed in instead of hard coded?